src/SFA.DAS.LevyTransferMatching.Web/Extensions/ApplicationViewModelExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SFA.DAS.LevyTransferMatching.Web.Models.Pledges;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Extensions
+{
+    public static class ApplicationViewModelExtensions
+    {
+        public static string GetMatchPercentageCss(this ApplicationViewModel viewModel)
```

Comment:
Does this need to be an extension method? We could just implement this on the viewmodel itself couldn't we?

Reply:
yes we can and I initially implemented as such. It was requested in first tech review by Conor to be moved out of there as Extension method. I have moved it in view model again now.